Problem1.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     int val;
+ *     ListNode next;
+ *     ListNode(int x) { val = x; }
+ * }
+ */
+/**
+Leet Code Submitted : YES
+Time Complexity : O(2N)
+Space Complexity : O(N)
+
+Use Stack to store the contents of the list and then pop each element to reverse the list. (Not Optimal)
+
+**/
+
+class Solution {
+    public ListNode reverseList(ListNode head) {
+        Stack<Integer> s1 =  new Stack<>();
+        
+        if(head == null)
+            return null;
+            
+        ListNode temp = head;
+        while(temp != null){
+            //s1.add(temp.val);
+            temp = temp.next;
```

Comment:
you should not comment this line in which you are adding value to stack

Reply:
Yes, I was playing around with it and it seems submitted that change.
Will fix it